tests/Tests.FeatureManagement/Tests.FeatureManagement.csproj

```diff
@@ -1,32 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
While we are at it, should we also add .NET6 to the targets since it is the latest LTS release of .NET.

Reply:
Yeah, it's a good idea. but that would be better in a separate PR to add .NET 6 targeting.